src/Honeybee_ Run Energy Simulation.py

```diff
@@ -1254,11 +1298,17 @@ def main(north, epwFileAddress, EPParameters, analysisPeriod, HBZones, HBContext
                     if schedule != None:
                         if schedule.upper() not in EPScheduleCollection: EPScheduleCollection.append(schedule)
                     else: needToWriteMixSched = True
-            if zone.mixAir == True:
```

Comment:
Looks like you are overwriting part of @chriswmackey's code? Chris can check and confirm. @antonszilasi you probably need to sync with master again and check the diff.